libs/sdk-core/src/breez_services.rs

```diff
@@ -1368,6 +1370,36 @@ impl BreezServices {
             },
         })
     }
+
+    pub async fn register_notifications(
+        &self,
+        platform: String,
+        token: String,
+    ) -> SdkResult<RegisterPaymentNotificationResponse> {
+        info!("Registering notification");
+
+        // TODO Make configurable?
+        let base_url = "https://notifier.breez.technology";
+        let url = format!("/api/v1/notify{base_url}?platform={platform}&token={token}");
+
+        let message = url.clone();
+        let sign_request = SignMessageRequest { message };
+        let sign_response = self.sign_message(sign_request).await?;
+
+        let subscribe_request = SubscribeNotificationsRequest {
+            url,
+            signature: hex::decode(sign_response.signature)
+                .map_err(|e| anyhow!("Failed to decode signature: {e}"))?,
+        };
```

Comment:
Perhaps this logic should go int the lsp_api?

Reply:
Makes sense, done.